opm/autodiff/BlackoilPropsAdFromDeck.cpp

```diff
@@ -928,5 +928,42 @@ BlackoilPropsAdFromDeck::BlackoilPropsAdFromDeck(const BlackoilPropsAdFromDeck&
     }
 
 
+    /// Obtain the scaled critical oil in gas saturation values.
+    /// \param[in]  cells  Array of cell indices.
+    /// \return Array of critical oil in gas saturaion values.
+    V BlackoilPropsAdFromDeck::scaledCriticalOilinGasSaturations(const Cells& cells) const {
+
+
+        assert(phaseUsage_.phase_used[BlackoilPhases::Gas]);
+        assert(phaseUsage_.phase_used[BlackoilPhases::Oil]);
+
+        const int n = cells.size();
+        V sgocr = V::Zero(n);
```

Comment:
I wonder about the variable naming: it is called sgocr, but later we extract Sogcr. Is there a reason for the difference (flipped o and g)?